src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LightCompiler.cs

```diff
@@ -2390,7 +2529,7 @@ private void CompileInvocationExpression(Expression expr)
 
             if (typeof(LambdaExpression).IsAssignableFrom(node.Expression.Type))
             {
-                var compMethod = node.Expression.Type.GetMethod("Compile", Array.Empty<Type>());
+                var compMethod = node.Expression.Type.GetMethod("Compile", new Type[0]);
```

Comment:
There were a bunch of cases where Array.Empty change was applied to one branch or another and I tried to keep the change.
I somehow missed this one.